eva-server/src/main/java/uk/ac/ebi/eva/server/repository/VariantEntityRepository.java

```diff
@@ -35,5 +33,8 @@
 
     List<VariantEntity> findByIds(String id);
 
-    List<VariantEntity> findByChrAndStartWithMarginAndEndWithMargin(String chr, int start, int end);
+    List<VariantEntity> findByChrAndStartWithMarginAndEndWithMargin(String chr, int start, int end,
```

Comment:
Maybe it's time to rename this to something that reflects its "public intention":

* There are a lot more arguments than the region now
* The margins are an implementation detail

Maybe something like "findByRegionAndComplexFilters"? The method should be documented to explain what the "complex filters" are exactly.